java/core/generate-test-sources-build.xml

```diff
@@ -6,6 +6,7 @@
         <arg value="--proto_path=${test.proto.dir}"/>
         <arg value="${protobuf.source.dir}/google/protobuf/map_lite_unittest.proto"/>
         <arg value="${protobuf.source.dir}/google/protobuf/unittest.proto"/>
+        <arg value="${protobuf.source.dir}/google/protobuf/unittest_features.proto"/>
         <arg value="${protobuf.source.dir}/google/protobuf/unittest_custom_options.proto"/>
         <arg value="${protobuf.source.dir}/google/protobuf/unittest_enormous_descriptor.proto"/>
         <arg value="${protobuf.source.dir}/google/protobuf/unittest_import.proto"/>
```

Comment:
These changes to the Maven build are from #16176 and can be rebased away once that is merged.